pkg/apps/index.html

```diff
@@ -23,6 +23,7 @@
     <title translate="yes">Applications</title>
     <meta charset="utf-8">
     <link href="apps.css" type="text/css" rel="stylesheet">
+    <link href="../../static/theme-overrides.css" type="text/css" rel="stylesheet">
```

Comment:
While writing the PR's description I just realized that using

```suggestion
    <link href="cockpit/static/theme-overrides.css" type="text/css" rel="stylesheet">
```
might work too. Not sure, just speculating because of https://github.com/cockpit-project/cockpit/blob/388c2f423a6d844ca2d1d9a55ded7086a4245ff1/pkg/static/login.html#L18